lib/convection/model/template/resource/aws_sqs_queue_policy.rb

```diff
@@ -8,9 +9,25 @@ class Resource
         # AWS::SQS::QueuePolicy
         ##
         class SQSQueuePolicy < Resource
+          extend Forwardable
+
           type 'AWS::SQS::QueuePolicy'
-          property :queue, 'Queues', :type => :list
-          property :policy_document, 'PolicyDocument'
+          property :queues, 'Queues'
```

Comment:
For reference:
`:queue` - the accessor in ruby
`Queues` - the Cloudformation JSON key
`:type` - data, defaults to `:string`